docs/getting_started.rst

```diff
@@ -37,17 +57,40 @@ Using WhyLogs API
 Initialize a Logging Session
 ########################
 
-TBD
+An example script for creating a logging session can be found `here <https://whylogs.readthedocs.io/en/latest/auto_examples/log_dataframe.html#script>`__.
 
 Create a Logger
 ########################
 
-TBD
+Loggers log statistical information about your data. They have the following parameters:
+
+- **dataset_name** sets the name of the dataset, to be used in DatasetProfile metadata and generated filenames.
+- **dataset_timestamp** sets start and end timestamps for the data.
+- **session_timestamp** sets start and end timestamps for the session.
+- **writers** provides a list of writers that will be used to create the DatasetProfile.
+- **verbose** sets the verbosity of the output.
+
+For more information, see the `documentation <https://whylogs.readthedocs.io/en/latest/autoapi/whylogs/app/logger/index.html>`_ for the logger class.
+
+`This example code <https://whylogs.readthedocs.io/en/latest/auto_examples/configure_logger.html>`_ uses logger options to control the output location. 
 
 Configure a Writer
 ########################
-TBD
+
+Writers write the statistics gathered by the logger into an output file. They have the following parameters:
+- **output_path** sets the location output files will be stored. Use a directory path if your writer ``type = 'local'``, or a key prefix for ``type = 's3'``.
```

Comment:
Need an empty line between the text and first bullet point for Markdown to format correctly.